bzt/resources/selenium_extras.py

```diff
@@ -297,67 +296,65 @@ def get_loop_range(start, end, step):
     return range(start, end, step)
 
 
-class FrameManager:
-
-    @staticmethod
-    def switch(frame_name=None):
-        driver = _get_driver()
-        try:
-            if not frame_name or frame_name == "relative=top":
-                driver.switch_to_default_content()
-            elif frame_name.startswith("index="):  # Switch using index frame using relative position
-                driver.switch_to.frame(int(frame_name.split("=")[1]))
-            elif frame_name == "relative=parent":  # Switch to parent frame of the current frame
-                driver.switch_to.parent_frame()
-            else:  # Use the selenium alternative
-                driver.switch_to.frame(frame_name)
-        except NoSuchFrameException:
-            raise NoSuchFrameException("Invalid Frame ID: %s" % frame_name)
-
-
-class WindowManager:
-    def __init__(self):
-        self.windows = {}
-
-    def switch(self, window_name=None):
-        driver = _get_driver()
-        try:
-            if not window_name:  # Switch to last window created
-                driver.switch_to.window(driver.window_handles[-1])
+def switch_frame(frame_name=None):
+    driver = _get_driver()
+    try:
+        if not frame_name or frame_name == "relative=top":
+            driver.switch_to_default_content()
+        elif frame_name.startswith("index="):  # Switch using index frame using relative position
```

Comment:
If I use `- switchFrameByName(frame-top)` command in YAML, I'll get an error: `AttributeError: 'WebElement' object has no attribute 'startswith'`.

In general, selenium allows `frame_name` to be:
1. integer index (supported in this method).
2. string name (not supported in this method) - optional, not sure we really need it.
3. web element (supported, but causes above error).